src/cdk-experimental/combobox/combobox.ts

```diff
@@ -188,10 +241,18 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
 
   private _coerceOpenActionProperty(input: string | OpenAction[]): OpenAction[] {
     let actions: OpenAction[] = [];
+    const viableActions = ['focus', 'click', 'downKey', 'toggle'];
+
     if (typeof input === 'string') {
-      actions.push(input as OpenAction);
+      const tokens = input.trim().split(/[ ,]+/);
+      for (const token of tokens) {
+        if (viableActions.indexOf(token) === -1) {
+          throw Error(`${token} is not a supported open action`);
+        }
+        actions.push(token as OpenAction);
+      }
     } else {
-      actions = input;
+      actions = coerceArray(input);
     }
     return actions;
```

Comment:
```ts
let actions = typeof input === 'string' ? input.trim().split(/[ ,]+/) : input;
if (isDevMode() && actions.some(a => allowedOpenActions.indexOf(a) === -1)) {
  throw Error(`${input} is not a support open action for CdkCombobox`);
}
return actions;
```
* We'd want to do the sanity check even if the value isn't a string
* We'd only want to do the sanity check in dev mode
* I think the `else` isn't doing anything since it's already an array if it's not a string

Reply:
I see this makes sense. The some function helps condense it too.